tests/ui/repr/repr-align.rs

```diff
@@ -9,9 +9,6 @@ struct S1(i32);
 #[repr(align(4294967296))] //~ ERROR: invalid `repr(align)` attribute: larger than 2^29
 struct S2(i32);
 
-#[repr(align(536870912))] // ok: this is the largest accepted alignment
-struct S3(i32);
-
```

Comment:
You should be able to leave these in the same file, just using revisions. Something like

```text
//@ revisions: win notwin
//@[win] only-windows
//@[notwin] ignore-windows
```

And then update the annotations to `//[win]~ ERROR`, `//[notwin]~ ERROR`, or `//[win,notwin]~ ERROR` based on when they should appear.